src/app/static/src/app/components/header/DataExplorationSupportMenu.vue

```diff
@@ -10,31 +11,79 @@
                          v-translate="'axe'">
             </router-link>
         </drop-down>
+        <error-report :open="errorReportOpen"
+                      @send="sendErrorReport"
+                      @close="toggleErrorReportModal">
+            <template v-slot:sectionView>
+                <div>
+                    <label for="section" v-translate="'section'"></label>
+                    <select class="form-control" v-model="section" id="section">
```

Comment:
Can drop `v-model` here - since there is only one option in this dropdown, the value can only ever be "dataExploration"!